_maps/map_files/RandomRuins/SpaceRuins/derelict5.dmm

```diff
@@ -1,1745 +1,2017 @@
 //MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
-"a" = (
-/turf/template_noop,
-/area/template_noop)
-"b" = (
-/turf/simulated/mineral,
+"aN" = (
+/obj/effect/spawner/lootdrop/crate_spawner,
+/obj/effect/decal/cleanable/dirt,
+/turf/simulated/floor/plasteel,
 /area/ruin/space/unpowered)
-"d" = (
-/turf/simulated/floor/plating/asteroid/airless,
+"bO" = (
+/mob/living/simple_animal/hostile/skeleton,
+/turf/simulated/floor/plasteel,
 /area/ruin/space/unpowered)
-"e" = (
-/turf/simulated/wall,
+"bQ" = (
+/obj/structure/table,
+/obj/item/stack/ore/gold{
+	pixel_x = 8
+	},
+/obj/item/stack/ore/gold{
+	pixel_y = -5
+	},
+/obj/item/stack/ore/gold{
+	pixel_y = 2
+	},
+/obj/item/stack/ore/gold{
+	pixel_x = -5
+	},
+/turf/simulated/floor/plasteel,
 /area/ruin/space/unpowered)
-"f" = (
-/obj/effect/spawner/random_spawners/wall_rusted_always,
-/turf/simulated/wall,
+"dz" = (
+/obj/structure/closet/crate,
+/obj/item/storage/bag/ore,
+/obj/effect/decal/cleanable/dirt,
+/turf/simulated/floor/plasteel,
 /area/ruin/space/unpowered)
-"g" = (
-/turf/simulated/floor/plating,
+"et" = (
+/mob/living/simple_animal/cockroach,
+/turf/simulated/floor/plasteel,
 /area/ruin/space/unpowered)
-"h" = (
+"fi" = (
+/obj/structure/closet/cabinet,
+/obj/item/paper{
+	name = "The Bunker Key";
+	info = "In a game of direction I hold the key, to find your way solve this riddle for me,<br>Three tiles east from the front door, stand still turn right and look straight forth,<br>Continue onward, touch the wall and you'll see, the bunker entrance is where you will be."
+	},
```

Comment:
don't manually var edit this. Instead, create a new type of away-mission paper and code it in

Reply:
> don't manually var edit this. Instead, create a new type of away-mission paper and code it in

done, it's been added under ruins in mission_code, thank you!